projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -323,19 +309,18 @@ export function checkAccountPageTemplateDisplayed() {
     'Verifies whether the account page template is displayed',
     checkAccountPageTemplateDisplayed.name
   );
-  cy.get(accountPageTemplateSelector).should('be.visible');
+  cy.get('.AccountPageTemplate').should('be.visible');
```

Comment:
This method is always called when we want to assert that the quote list page is displayed. So instead of checking for the `.AccountListTemplate`, we should better check for the quoteListComponentSelector. Of course one should also rename the method accordingly